aineko/extras/connectors/wss.py

```diff
@@ -0,0 +1,209 @@
+# Copyright 2023 Aineko Authors
+# SPDX-License-Identifier: Apache-2.0
+"""Extra module for connecting to a WebSocket."""
+
+import json
+import time
+from typing import Any, Dict, List, Optional
+
+import websocket
+from pydantic import BaseModel, field_validator
+
+from aineko import AbstractNode
+from aineko.utils.secrets import inject_secrets
+
+
+class ParamsWSS(BaseModel):
+    """Connector params for WebSocket model."""
+
+    max_retries: int = 30
+    retry_sleep: float = 5
+    url: str
+    header: Optional[Dict[str, str]] = None
+    init_messages: List[Any] = []
+    metadata: Optional[Dict[str, Any]] = None
+
+    @field_validator("url")
+    def supported_url(cls, u: str) -> str:  # pylint: disable=no-self-argument
```

Comment:
Single character names should be avoided.

Reply:
fixed here: https://github.com/aineko-dev/aineko/pull/121/commits/7eb4998eee7e29d284e76097c5f12d44c84e7e78